ui.apps/src/main/content/jcr_root/apps/core/fd/components/formsportal/link/v1/link/link.html

```diff
@@ -0,0 +1,30 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2021 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<sly data-sly-use.commonsTemplates="core/wcm/components/commons/v1/templates.html"
+     data-sly-use.component="com.adobe.cq.wcm.core.components.models.Component"
+     data-sly-use.link="com.adobe.cq.forms.core.components.models.formsportal.Link"
+     data-sly-test.hasText="${link.linkText}">
+    <a
+        id="${component.id}"
+        class="cmp-link coral-Link"
```

Comment:
I believe we should wrap it under a div to have more control ?. Probably a markup similar as this, [1]

[1] https://aemcomponents.dev/content/core-components-examples/library/page-authoring/embed.html